src/azurelargeinstance/azext_azurelargeinstance/aaz/latest/large_instance/_update.py

```diff
@@ -15,16 +15,17 @@
     "large-instance update",
 )
 class Update(AAZCommand):
-    """Update the Tags field of an Azure Large Instance for the specified subscription, resource group, and instance name.
+    """Update the Tags field of an Azure Large Instance for the specified
+subscription, resource group, and instance name.
 
     :example: To add an Azure Large Instance tag
-        az large-instance update --subscription $SUBSCRIPTION_ID --instance-name=$INSTANCE_NAME --resource-group=$RESOURCE_GROUP --tags newKey=value
+        az large-instance update --subscription $SUBSCRIPTION_ID --instance-name=$INSTANCE_NAME --resource-group=$RESOURCE_GROUP --tags
```

Comment:
You forget the tags value for updates. BTW in most of examples, the `--subscription` argument is optional because cli core will use the default subscription as value after they login
```suggestion
        az large-instance update --subscription $SUBSCRIPTION_ID --instance-name=$INSTANCE_NAME --resource-group=$RESOURCE_GROUP --tags newKey=value
```